### ISSUE TITLE
Should excerpt be private

### ISSUE THREAD

**Issue by samikeijonen** [samikeijonen/edd-members#3]
Should we make an option to make excerpt private? Now they are never private.

**Comment by tnorthcutt** [comment]
On the one hand, I like the WP philosophy of "decisions over options."

On the other hand, this does seem like a pretty simple option to add. As a user, I _think_ I would normally expect **everything** to be private by default, with the option to make excerpts public. But, of course, it's hard to make a change like that to the plugin once people are already using it :smile: 

So, consider this a vote for adding an option to make excerpts private.

**Comment by samikeijonen** [comment]
At the moment I have made a decision that only singular post type can be private. Usually excerpts are used in Blog or archive pages.

I'll leave this open and re-think about this.

**Comment by tnorthcutt** [comment]
I think that makes sense (only allowing `is_singular()` to be set private). Right now, would that allow the excerpt from a (private) single post to be shown on an archive page? I think that would be confusing to a user, since they will have marked a post as private, but there could still be a way to see that post.

On the other hand, it's a good way of having private content, but being able to show a "teaser" of that content, which is often desired on membership sites.

**Comment by samikeijonen** [comment]
Yes excerpt  would show up in archive pages like the content also. So if `the_content()` function is used in archive pages also, writer should use the “Read More” tag. Before that content would be a teaser content.
